compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -529,3 +571,249 @@ def _process_obs(self, curr_obs, sequence_example):
           curr_obs_feature, dtype=obs_dtype, name=curr_obs_feature_name)
       add_feature_list(sequence_example, curr_obs_feature,
                        curr_obs_feature_name)
+
+
+class ModuleWorkerUtility:
+  """Utility class to process ExplorationWorker results for ModuleWorker."""
+
+  def __init__(self, base_path: Optional[str] = None):
+    self._base_path = base_path
+
+  def partition_for_loss(self,
```

Comment:
does this need to be public, or is it always what happens after `process_succeeded`?

Reply:
It's getting called in ModuleWorker, not sure how I can make it private.

Reply:
is it basically an implementation detail of `process_succeeded`?

Reply:
We could potentially move it to ModuleWorker and make it private there.